packages/checkout-full/core/src/components/malga-checkout-full/partials/malga-checkout-full-identification/malga-checkout-full-identification.tsx

```diff
@@ -159,6 +172,7 @@ export class MalgaCheckoutFullIdentification {
 
   render() {
     const documentTypesByCountries = documentTypesByCountry(this.locale)
+    const isTheSelectedCountryBR = this.formValues.country === 'BR'
```

Comment:
Aqui tenho uma dúvida pra ti e pro @filipperodrigues, será que não era interessante incrementar essa lógica para caso o cliente seja nacional, a gente ja botar como padrão o campo de Estado como um SelectField?

Algo como isso:

```suggestion
    const isTheSelectedCountryBR = this.formValues.country === 'BR' || !this.internationalCustomer
```

O único contra que vejo nisso é no caso do cliente do nosso cliente ser BR mas o endereço ser internacional, mas acredito que esse cenário não existe, a porque aí o uso do checkout teria que ser com o `internationalCustomer` como `true`. Faz sentido?

Reply:
Na minha opinião não deveria existir, o recomendado nesses casos seria usar o checkout com a versão internacional. Porque o modelo de negócia da empresa do nosso cliente seria muito estranho, seriam vendas pra brasileiros que moram em outros países. Mas levantei mais como reflexão, eu posso estar full enviesado com essa suposição